### PR TITLE
Add extra new line between separate queries

### PR DESCRIPTION
**Pull request by mvorisek**
impl. #53, I like it!

### CODE REVIEW

File: tests/format-highlight.html [hunk 1]
```diff
@@ -798,6 +798,7 @@
   <span >*</span>
 <span style="font-weight:bold;">LIMIT</span>
   <span style="color: green;">1</span><span >;</span>
+
```
Comment (mvorisek) [inline tests/format-highlight.html:801]: we need to preserve NL in HTML render

File: src/HtmlHighlighter.php [hunk 2]
```diff
@@ -56,6 +56,7 @@ public function highlightToken(int $type, string $value): string
         return '<span ' . $attributes . '>' . $value . '</span>';
     }
 
+    /** @param Token::TOKEN_TYPE_* $type */
```
Comment (SenseException) [inline src/HtmlHighlighter.php]: Maybe those improvements should happen in a separate PR.

Comment (mvorisek) [reply src/HtmlHighlighter.php]: Sure, moved to #125.

**Comment by mvorisek** [comment]
rebased

**Comment by greg0ire** [comment]
I think there should be a test showing that this avoids the `; CREATE TABLE` weirdness from the referenced issue.

**Comment by mvorisek** [comment]
> I think there should be a test showing that this avoids the `; CREATE TABLE` weirdness from the referenced issue.

We already test there is new line after `;`, especially after #110 fix which imply such testing already - is this reply enough?

**Comment by greg0ire** [comment]
So, #110 already fixes (the most important) part of #53, is that it?

**Comment by mvorisek** [comment]
> So, #110 already fixes (the most important) part of #53, is that it?

Yes - #53 is no longer a bug, it is a feature request to add 2 new lines (1 empty) between two queries. This is what this feature does.

**Review by greg0ire — APPROVED** [review]
Why not…

**Review by SenseException — APPROVED** [review]

**Pull request merged**